students/sikorski_marcin/Lesson_1_-_Python_basics.py

```diff
@@ -0,0 +1,128 @@
+#1 Sum of three numbers
+
+a = int(input())
+b = int(input())
+c = int(input())
+print(a+b+c)
+
+#2 Area of right-angled triangle
+a = int(input())
+b = int(input())
+print(0.5*a*b)
+
+#3 Hello, Harry!
+
+a = input()
+print("Hello, "+ a + "!")
+
+#4 Apple sharing
+a = int(input())
+b = int(input())
+print(b//a)
+print(b%a)
+
+#5 Previous and next
+a = int(input())
+print('The next number for the number ' + str(a)  + " is " + str(a+1) + ".")
+print('The previous number for the number ' + str(a)  + " is " + str(a-1) + ".")
+
+#6 School desks
+a = int(input())
+b = int(input())
+c = int(input())
+
+l = [a,b,c]
+sum = 0
+for x in l:
```

Comment:
można to znacznie prościej zrobić wykorzystując tylko '+' i "//" lub skorzystać z funkcji 'ceil' z modułu 'math'.
Na tą chwilę nie jest konieczne wykorzystywanie pętli i listy.